frontend/routes/viewerGui/components/groups/groups.component.tsx

```diff
@@ -182,16 +193,17 @@ export class Groups extends React.PureComponent<IProps, IState> {
 
 	public renderListView = renderWhenTrue(() => (
 		<>
-			<ViewerPanelContent>
+			<ViewerPanelContent onClick={this.handleClickOutside}>
 				{this.renderEmptyState(!this.props.searchEnabled && !this.state.filteredGroups.length)}
 				{this.renderNotFound(this.props.searchEnabled && !this.state.filteredGroups.length)}
 				{this.renderGroupsList(this.state.filteredGroups.length)}
```

Comment:
Instead of referencing each individual component, the propagation of the click could be handled here by wrapping the components and handling the click.
```suggestion
<div onClick={(e:MouseEvent) => e.stopPropagation()}>
				{this.renderEmptyState(!this.props.searchEnabled && !this.state.filteredGroups.length)}
				{this.renderNotFound(this.props.searchEnabled && !this.state.filteredGroups.length)}
				{this.renderGroupsList(this.state.filteredGroups.length)}
</div>
```
Same goes with reportedItems.